public/app/core/components/AppChrome/MegaMenu/MegaMenu.tsx

```diff
@@ -64,6 +64,10 @@ export const MegaMenu = memo(
     const onPinItem = (id?: string) => {
       if (id && config.featureToggles.pinNavItems) {
         const newItems = isPinned(id) ? pinnedItems.filter((i) => id !== i) : [...pinnedItems, id];
+        const interactionName = isPinned(id) ? 'grafana_nav_item_unpinned' : 'grafana_nav_item_pinned';
```

Comment:
Put `isPinned(id)` in a variable so we don't call it twice in this line and the previous line